app/src/main/java/com/nervousfish/nervousfish/activities/QRExchangeKeyActivity.java

```diff
@@ -148,25 +147,27 @@ private void addNewContact(final String qrMessage) {
 
 
     /**
-     * Shows the QR Code in an alert dialog popup screen
-     *
-     * @param qrCode The QR code to be shown.
+     * Shows the QR Code in the activity.
      */
     @SuppressLint("InflateParams")
-    private void showQRCode(final Bitmap qrCode) {
+    private void showQRCode() {
 
-        final LayoutInflater li = LayoutInflater.from(this);
-        final View myView = li.inflate(R.layout.qrcode, null);
+        final String space = " ";
+        final Bitmap qrCode = QRGenerator.encode(publicKey.getType() + space + publicKey.getName()
+                + space + publicKey.getKey());
+//        final LayoutInflater li = LayoutInflater.from(this);
+//        final View myView = li.inflate(R.layout.qrcode, null);
 
-        final ImageView imageView = (ImageView) myView.findViewById(R.id.QR_code_image);
+        final ImageView imageView = (ImageView) this.findViewById(R.id.QR_code_image);
         imageView.setImageBitmap(qrCode);
-        final AlertDialog.Builder builder = new AlertDialog.Builder(this)
-                .setView(imageView)
-                .setPositiveButton(getString(R.string.done), new QRCloser());
-
-        ((ViewGroup) imageView.getParent()).removeView(imageView);
-        lastDialog = builder.create();
-        lastDialog.show();
+        imageView.setMaxWidth(30);
+//        final AlertDialog.Builder builder = new AlertDialog.Builder(this)
+//                .setView(imageView)
+//                .setPositiveButton(getString(R.string.done), new QRCloser());
+//
+//        ((ViewGroup) imageView.getParent()).removeView(imageView);
+//        lastDialog = builder.create();
+//        lastDialog.show();
```

Comment:
Can you remove commented code 😇 

Also can you removed the unused code on [line 174](https://github.com/ericcornelissen/NervousFish/pull/224/files#diff-3ee102e75053006952e1d321d14f640dR174)